builds/e2e/connectivity.yaml

```diff
@@ -9,10 +9,10 @@ resources:
   pipelines:
   - pipeline: images
     source: 'Azure-IoT-Edge-Core Build Images'
-    branch: 'master'
+    branch: 'release/1.1'
```

Comment:
You may want to point out in the PR description that this is for manual builds only (and scheduled, but as you pointed out, scheduled triggers aren't set up in this branch).